runtime/doc/options.txt

```diff
@@ -2184,6 +2184,26 @@ A jump table for the options with a short description can be found at |Q_op|.
 	This option is reset when the 'paste' option is set and restored when
 	the 'paste' option is reset.
 
+				*'filechangenotify'* *'fcnotify'*
+'filechangenotify' 'fcnotify'	string (default "off")
+				local to buffer
+	Watch for changes to a file. How to react for the changes depends on
+	the value of the option. The option can take a list of following values:
+
+		off		Don't react to changes happening to a file.
+				Can't be set with anyother of the following
+				values.
```

Comment:
This should probably just be "Cannot be set with any other value"

Reply:
Or better "Cannot be combined with any other value."